src/fireblocks-sdk.ts

```diff
@@ -89,6 +91,31 @@ export class FireblocksSDK {
         return await this.apiClient.issueGetRequest(url);
     }
 
+    /**
+     * Gets a list of vault accounts per page matching the given filter or path
+     * @param nextOrPreviousPageUrl URL containing the full path for the previous/next page as returned from previous request result
+     */
+    public async getVaultAccountsWithPageInfo(nextOrPreviousPageUrl: string): Promise<PagedVaultAccountsResponse>;
+    /**
+     * Gets a list of vault accounts per page matching the given filter or path
+     * @param pagedVaultAccountsRequestFilters Filters for the first request
+     */
+    public async getVaultAccountsWithPageInfo(pagedVaultAccountsRequestFilters: PagedVaultAccountsRequestFilters): Promise<PagedVaultAccountsResponse>;
+    public async getVaultAccountsWithPageInfo(input: string | PagedVaultAccountsRequestFilters): Promise<PagedVaultAccountsResponse> {
+        if (typeof input === "string") {
```

Comment:
consider applying with some regex to validate that this value satisfy the expectation (e.g.: Url full format...)